components/core/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ endif()
 if(CLP_USE_STATIC_LIBS)
     set(Boost_USE_STATIC_LIBS ON)
 endif()
-find_package(Boost REQUIRED iostreams program_options filesystem system)
+find_package(Boost 1.74 REQUIRED iostreams program_options filesystem system)
```

Comment:
In `install-boost.sh`, we're also installing `regex`. Is that (still) necessary?